test/expected/hnsw_todo.out

```diff
@@ -38,14 +38,15 @@ ERROR:  relation "small_world_l2_vector_int_idx" does not exist at character 41
 EXPLAIN (COSTS FALSE)
 SELECT id, ROUND(l2sq_dist(vector_int, array[0,1,0])::numeric, 2) as dist
 FROM small_world_l2
-ORDER BY vector_int <-> array[0,1,0] LIMIT 7;
-                               QUERY PLAN                               
-------------------------------------------------------------------------
+ORDER BY vector_int <?> array[0,1,0] LIMIT 7;
+                              QUERY PLAN                               
+-----------------------------------------------------------------------
  Limit
-   ->  Sort
-         Sort Key: (public.l2sq_dist(vector_int, '{0,1,0}'::integer[]))
-         ->  Seq Scan on small_world_l2
-(4 rows)
+   ->  Result
+         ->  Sort
+               Sort Key: (l2sq_dist(vector_int, '{0,1,0}'::integer[]))
```

Comment:
hm, do you know why before the Sort Key function was schema-qualified (`public.l2sq_dist`) while now it is not (`l2sq_dist`) ?

Reply:
Actually that was added after the latest release, before that it was without public if I remember correctly, not sure why but one change that I did to that operator was adding it for integer type as well